### PR TITLE
Adds ui for admin/modules

### LINKED ISSUE #184: Add admin/modules page (UI only)
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/open-event-frontend

screenshot from 2017-06-08 07-34-15

Located at `admin/modules`

I would we working on this.

### PR DESCRIPTION
**Pull request by geekyd**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Adds ui for modules sub-route in admin route.

Deployment link: https://open-event-frontend-app2.herokuapp.com/admin/modules
Fixes #184 

@niranjan94 @CosmicCoder96 Please review. Thanks

### CODE REVIEW

**Comment by utkarshnath** [comment]
@geekyd Don't you think that there is too big a gap between the text and the checkbox?

**Comment by geekyd** [comment]
@utkarshnath Thanks for pointing that out. The idea here was to provide a consistent design language, please check https://open-event-frontend-app2.herokuapp.com/settings/email-preferences I think we must follow a common design language throughout the app. If you have a better solution, we can go with it. :)

**Comment by geekyd** [comment]
@niranjan94 Please guide regarding this. Thanks!

File: app/templates/admin/modules.hbs [hunk 1]
```diff
@@ -1 +1,36 @@
-{{outlet}}
+<h3 class="ui header">
+  {{t 'Enable/Disable Modules'}}
```
Comment (niranjan94) [inline app/templates/admin/modules.hbs]: This header can be removed

File: app/templates/admin/modules.hbs [hunk 2]
```diff
@@ -1 +1,36 @@
...
+</h3>
+<form class="ui form">
+  <div class="ui grid">
+    <div class="row">
```
Comment (niranjan94) [inline app/templates/admin/modules.hbs]: Put the contents inside a segment and make it look like the email preferences.

Comment (geekyd) [inline app/templates/admin/modules.hbs]: @niranjan94 Done.

**Comment by geekyd** [comment]
@niranjan94 Please review. Here is a screenshot of the ui
screenshot from 2017-06-08 19-53-12

File: app/templates/admin/modules.hbs [hunk 3]
```diff
@@ -1 +1,33 @@
-{{outlet}}
+<form class="ui form">
+  <div class="ui vertical segment grid">
```
Comment (niranjan94) [inline app/templates/admin/modules.hbs]: make this a basic segment. This page is anyway wrapped in a segment by the parent route.

**Comment by geekyd** [comment]
@niranjan94 Please review. Thanks

**Pull request merged**